Source/Scene/Model.js

```diff
@@ -856,7 +853,6 @@ define([
         var sceneOffset = byteOffset;
         var binOffset = sceneOffset + sceneLength;
 
-        // If sceneFormat isn't 0 (JSON) then we are using the CESIUM_binary_glTF extension
         // This means the last 2 integers of the header are jsonOffset & jsonLength
         if (sceneFormat !== 0) {
```

Comment:
Since we no longer support CESIUM_binary_glTF, if `sceneFormat` isn't 0, then we don't support the format and should throw an error instead of changing these values.